### PR TITLE
12.10.0 version bump and release notes.

### PR DESCRIPTION
**Pull request by marcparadise**
Pending:

* [x] Updated component list in changelog, changelog generation failed to find prior version manifest.

Signed-off-by: Marc Paradise <marc@chef.io>

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -7,6 +7,39 @@ in the release. For a detailed list of changed components, refer to
...
+* Upgrade to OpenSSL 1.0.2j.  The prior release (1.0.1u) is approaching EOL.
+* Updated TLS ciphers. See compatibility notes, below.
+
+### Comaptibility Notes
+
+1. The change of TLS ciphers can cause older tooling to fail to negotiate
+   SSL sessions with the Chef Server. The changes to the cipher list are
+   captured [here](https://github.com/chef/chef-server/pull/918#issuecomment-244430458).
+   Upgrading any custom clients of the Chef Server API to use a current SSL
+   release will resolve this.
```
Comment (stevendanna) [inline RELEASE_NOTES.md]: Another thing they could do is set the cipher suite to a more compat-friendly version in their chef-server.rb

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -7,6 +7,39 @@ in the release. For a detailed list of changed components, refer to
...
+  correct source.
+- Regression fix: that caused errors on reconifgure when LDAP bind password
+  is nil has been fixed.
+
+### Security Updates
+
+* Upgrade to OpenSSL 1.0.2j.  The prior release (1.0.1u) is approaching EOL.
+* Updated TLS ciphers. See compatibility notes, below.
+
+### Comaptibility Notes
```
Comment (tas50) [inline RELEASE_NOTES.md]: Typo

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -7,6 +7,39 @@ in the release. For a detailed list of changed components, refer to
...
+### Chef Server
+- Smaller download - the download size has been reduced by around 35% via removal of redundant, cached, and unused components. The installed size has been similarly reduced.
+- add retry support to opscode-expander
+- `chef-server-ctl reindex` will now continue even if some objects are
+  not indexable, and will show which objects failed at the conclusion
+  of the run.
+- Data Collector support for Policyfiles.
+- `chef-server-ctl install` add-on installation now pulls from the
+  correct source.
+- Regression fix: that caused errors on reconifgure when LDAP bind password
```
Comment (tas50) [inline RELEASE_NOTES.md]: reconifgure typo

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -7,6 +7,39 @@ in the release. For a detailed list of changed components, refer to
...
+* Updated TLS ciphers. See compatibility notes, below.
+
+### Comaptibility Notes
+
+1. The change of TLS ciphers can cause older tooling to fail to negotiate
+   SSL sessions with the Chef Server. The changes to the cipher list are
+   captured [here](https://github.com/chef/chef-server/pull/918#issuecomment-244430458).
+   Upgrading any custom clients of the Chef Server API to use a current SSL
+   release will resolve this.
+2. With this TLS cipher suite change, the Reporting add-on will report
```
Comment (stevendanna) [inline RELEASE_NOTES.md]: Not sure we want to include it here, but all it takes to fix it is adding:

```
ssl_version :TLSv1_2
```

to `/etc/opscode-reporting/pedant_config.rb`

Comment (ksubrama) [inline RELEASE_NOTES.md]: I think you already helped me do that - https://github.com/chef/oc_reporting/commit/7c5a2b27d2f81e3e3b0bd0fd2ee5bc3258d1e13a

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,90 @@
...
+* config\_guess (9152ce40 -> 084a4076)
+* openssl (1.0.1u -> 1.0.2j)
+* mixlib-install (b2495ce9 -> 9c9dad45)
+* openresty (1.9.7.2 -> 1.11.2.1)
+* knife-opc (2ce18240 -> a08237d4)
+* ohai (567dcb84 -> 582dcc7a)
+* appbundler (379a06cc -> 76cb1728)
+* rb-readline (cf67cd06 -> 5e1c55db)
+* chef_backup-gem (cc8e5bf9 -> 67b1f51a)
+* veil-gem (master -> master)
```
Comment (stevendanna) [inline CHANGELOG.md]: Hrm, it is interesting that it didn't pick up the sha's here...Not to worried about it for now but might be worth looking at later.

**Pull request merged**